### ISSUE TITLE
Empty output.zip

### ISSUE THREAD

**Issue by ingenieroariel** [SEL-Columbia/modelrunner#109]
After successfully getting a job to the PROCESSED stage the resulting output.zip was completely empty. I used the `electrificationplanner` model from master.

What are the options to debug such runs? 

(My next step is to try to run the model sh file directly on the job folder and do step by step)
-a

**Comment by chrisnatali** [comment]
If you have ModelRunner up and running already, the easiest option is to just check the log via the web interface.  Click the Log link on the `/jobs` page for the job you want to debug:
![2018-07-19-134540_511x41_scrot](https://user-images.githubusercontent.com/740068/42960536-19de810c-8b5a-11e8-92b7-809316b4e861.png)

That pulls the log directly from the worker if the job is still running OR from the primary server if the job has completed or failed.  You should see output like the following:

```
Running electrificationplanner
Reading input from /home/mr/modelrunner/worker_data/3a40c5cb-51bc-4512-9da5-2bf682cae464/input
~/infrastructure-planning ~/modelrunner
Traceback (most recent call last):
  File "estimate_electricity_cost_by_technology_from_population.py", line 300, in <module>
    ], x.parse_args().__dict__, FULL_KEYS)
  File "/home/mr/infrastructure-planning/infrastructure_planning/macros.py", line 56, in load_and_run
    g = load_files(g)
  File "/home/mr/infrastructure-planning/infrastructure_planning/parsers.py", line 27, in load_files
    value = load_geotable(v)
  File "/home/mr/infrastructure-planning/infrastructure_planning/parsers.py", line 48, in load_geotable
    proj4, geometries, fields, definitions = geometryIO.load(path)
  File "<decorator-gen-4>", line 2, in load
  File "/home/mr/miniconda/envs/electrificationplanner/lib/python2.7/site-packages/archiveIO-0.4.4-py2.7.egg/archiveIO/__init__.py", line 176, in load
IOError: Could not open archive
~/modelrunner
```

To go directly to that log, you can find the file on the worker server under `/home/mr/modelrunner/worker_data/<job-id>/job_log.txt` where modelrunner puts all the output from the model job.  You'll need to know the id of the job which you can get from hovering over the Log link on the web-interface.  

You can also list all jobs via running the job_list script from an activated modelrunner conda environment:

```
python scripts/job_list.py
worker_url,worker_data_dir,primary_url,on_primary,primary_data_dir,name,status,uuid,model,created
http://xx.xx.xxx.xx:8888,worker_data,http://yy.yyy.yyy.yyy:8000,False,data,EA-Test-LeonaOutputs-03-existingMVlines,RUNNING,71330714-c679-49cb-95f5-4bc208fd46f1,electrificationplanner,2018-07-10T10:04:59.359840
http://xx.xx.xxx.xx:8888,worker_data,http://yy.yyy.yyy.yyy:8000,False,data,NP_MR_Inputs-PastProv-500kWh-03-LinesExisting,RUNNING,37546264-e248-4633-bf40-e8609327601b,electrificationplanner,2018-07-10T09:33:37.840959
http://xx.xx.xxx.xx:8888,worker_data,http://yy.yyy.yyy.yyy:8000,True,data,EA-Test-PastProv-02,COMPLETE,213b6810-98c5-40b2-9905-88f22b04f45a,electrificationplanner,2018-07-10T09:27:36.526302

```

**Comment by chrisnatali** [comment]
Otherwise, you can setup electrificationplanner to run locally.  You can probably use something similar to the [script](https://github.com/SEL-Columbia/modelrunner/blob/master/devops/setup_electrificationplanner.sh) modelrunner uses to set it up on a worker.

**Comment by ingenieroariel** [comment]
There was no error in the log when it failed (failed silently). What I ended up doing was running the worker process on a tmux window instead of starting it with fabric and figured out there were some problems with one of the dependencies.

Thanks for all the info. Closing the issue.

Unrelated: When the job is in PROCESSED but not COMPLETE there are no log or download links which complicates debugging a bit.
